README.md

```diff
@@ -1335,7 +1335,7 @@ It supports the following Spark deploy modes:
 
 ### Configuring Spark Project
 
-When using Spark backend uncomment the following Spark configuration in the application.conf file
+When using Spark backend uncomment the following Spark configuration in the reference.conf file, which is available under `core/src/resources`
```

Comment:
oh no, I think we should leave the conf files as they are. It's just this sentence that should say "move" instead of "uncomment"